### PR TITLE
Fix some typos

### PR DESCRIPTION
**Pull request by MicBruz**
I've fixed some typos.

### CODE REVIEW

**Comment by MicBruz** [comment]
Regarding failing CI:
I can see that the Legacy Ruby 9.1.17.0 build does not pass however it seems to be a different issue and not connected to my changes (the same issue is with different merged PRs). I don't know if you prefer to wait until it is fixed or not, so I decided to create that PR ;)

File: spec/rspec/core/configuration_spec.rb [hunk 1]
```diff
@@ -1793,7 +1793,7 @@ def metadata_hash(*args)
         expect(config.full_backtrace?).to eq true
       end
 
-      it 'returns false when backtrace patterns isnt empty' do
+      it "returns false when backtrace patterns aren't empty" do
```
Comment (JonRowe) [inline spec/rspec/core/configuration_spec.rb:1796]:
```suggestion
      it "returns false when backtrace patterns are not empty" do
```
Contractions tend to be unclear for non-native english speakers, whilst you are here do you mind removing it?

**Comment by JonRowe** [comment]
Don't worry about the legacy JRuby build, but just a minor suggestion

File: spec/rspec/core/configuration_spec.rb (hunk 1 above)
Comment (MicBruz) [reply spec/rspec/core/configuration_spec.rb]: Good point, I didn't know that. Thank you :)

**Pull request merged**